lib/access_import/accessors/accessor.rb

```diff
@@ -16,11 +16,17 @@ def add_other_accessors(accessors)
     end
   end
 
+  def add_records_hash
+    @records = hash_all unless @records
```

Comment:
You can store the values on the class:

```ruby
def self.all
  @all ||= hash_all
end
```

When calling from an instance-context call it like `Accessor.all`.